lib/tmdb/client.rb

```diff
@@ -11,6 +11,8 @@ class Error < StandardError; end
 
     class << self
       def request(endpoint, params)
+        return { results: [] } if params.blank?
+        return { results: [] } if params[:query] == ''
         return { results: [] } if params[:query].present? && searchable_query(params[:query]).empty?
```

Comment:
This is the client logic to resolve [TMDB-MOVIEQUEUE-6Z](https://mike-vallano.sentry.io/issues/3901921697/?project=127874&query=is%3Aunresolved&referrer=issue-stream) error when a user submits an empty string. It's kind of weird to have 3 lines that all return the same thing. Any ideas on how to make this prettier and still easy to understand?

Reply:
Since `searchable_query` returns nil if query isn't present, I think we could just do one line with:

`return { results: [] } if params.blank? || searchable_query(params[:query]).blank?`

Reply:
I'm not sure that does everything we need it to. It may the the case that the params are blank -- which would look like `{}`. It may be a separate case where the params aren't blank, but the specific value for the keyword `query` is blank -- which would look like `{ query: '' }`. The third case we're guarding against is if the `params[:query]` is actually full of symbols, which are not searchable.

Reply:
well, i tried it and your tests passed 😆 . 

i guess we'd need a test case for the scenarios you're talking about, and then we can tweak the logic to see if the tests pass 🤷

Reply:
to clarify what we talked about, I pulled this branch and updated the code to the one line I suggested above, then ran the tests for the file on this branch, and they passed. 

so it seems like either the one line suggested above would work, or we need additional test cases 😅